WalletWasabi/Mono/CommandSet.cs

```diff
@@ -450,10 +450,6 @@ public async Task<int> RunAsync(IEnumerable<string> arguments)
 				{
 					return await Help.InvokeAsync(extra);
 				}
-				if (arguments.All(x => !x.Contains("version")))
```

Comment:
What's this change?

Reply:
This change removes a hack used to handle an especial case. In this PR `--version` is not the only available parameter because we are adding `--datadir` too. I have to see how `Mono.Options` handles this shared options.

Reply:
I'm not worried about improperly working software when arguments are given inconsistently. I am worried about improperly working software when the arguments are correct. Get back to this issue when you checked that.